synapse/models/math.py

```diff
@@ -0,0 +1,50 @@
+import synapse.lib.module as s_module
+
+class MathModule(s_module.CoreModule):
+
+    def getModelDefs(self):
+        return (('math', {
+            'types': (
+
+                ('math:algorithm', ('guid', {}), {
+                    'doc': 'A mathematical algorithm.'}),
+
+                ('math:algorithm:type:taxonomy', ('taxonomy', {}), {
+                    'interfaces': ('meta:taxonomy',),
+                    'doc': 'A hierarchical taxonomy of algorithm types.'}),
+            ),
+            'edges': (
```

Comment:
crypto:algorithm -(uses)> math:algorithm?

Reply:
added a story to deprecate crypto:algorithm